### ISSUE TITLE
Default license

### ISSUE THREAD

**Issue by H-Mateus** [H-Mateus/dri-reproducibility-template#5]
Having a good default license as part of the template would probably be wise. 

Licensing isn't well thought about in the life sciences from my experience, so having a simple open license by default could be helpful.

I tend to prefer share-alike licences, and people will want attribution, so maybe either a [CC BY-SA](https://creativecommons.org/licenses/by-sa/4.0/) or just a [CC BY](https://creativecommons.org/licenses/by/4.0/) licence could be good candidates 🤔

**Comment by jatkinson1000** [comment]
If you want the code to be used by others then a slightly more permissive license than GPL/SA might be useful to encourage uptake (at the expense of the utopian vision).
MIT is a popular choice - it allows users to use the code commercially if they so wish.

I usually see CC licenses associated more with materials instead of code, but there's nothing saying they can't be used.

https://choosealicense.com/ is a useful resource, you could point people at this.

**Comment by H-Mateus** [comment]
Yeah, I like that CC licenses are modular, so if a user did want to tweak it so add SA or whatever else they could do so easily

My main beef with MIT is the allowance for closing the source, which just seems too antithetical to open and reproducible science for me, but I agree insisting on SA could push people away so that's probably better not being the default

SA and BY are probably the ones people will care about the most, so a brief explainer on those and a pointer to https://choosealicense.com/ in the docs would probably be a good start

The bosses at the DRI may have other priorities, so I'll bring it up with them when they fork this repo 🤷
